adapta/storage/delta_lake/README.md

```diff
@@ -22,6 +24,32 @@ adls_path = AdlsGen2Path.from_hdfs_path('abfss://container@account.dfs.core.wind
 # get Iterable[pandas.DataFrame]
 batches = load(azure_client, adls_path, batch_size=1000)
 ```
+
+### For S3 AWS Datalake Gen2
+
+```python
+import os
+from adapta.security.clients import AwsClient
+from adapta.security.clients.aws._aws_credentials import EnvironmentAwsCredentials
+from adapta.storage.delta_lake import load
+
+# Set up environment variables
+os.environ["PROTEUS__AWS_ACCESS_KEY_ID"] = minio_access_key_id
+os.environ["PROTEUS__AWS_SECRET_ACCESS_KEY"] = minio_secret_key
+os.environ["PROTEUS__AWS_REGION"] = "eu-central-1"
+os.environ["PROTEUS__AWS_ENDPOINT"] = "http://my-endpoint.com"
+
+credentials = EnvironmentAwsCredentials()
+aws_client = AwsClient(credentials)
+aws_client.initialize_session()
+
+blob_path = "s3a://dev/path/to/my/table"
+s3_path = S3Path.from_hdfs_path(blob_path)
+
+# get Iterable[pandas.DataFrame]
+batches = load(aws_client, s3_path, batch_size=1000))
```

Comment:
you need to print the result in the readme, otherwise nobody can verify their code

I'd advise adding a table creation from pandas Dataframe before, so you can showcase both write and read

Reply:
I've added a simple print for now, but I'm working on adding a creation from pandas Dataframe before. Since the Delta Lake load is expecting some logs I'm trying to get my head around it, but I'll update here once I'm done with it.

Reply:
After [2359273](https://github.com/SneaksAndData/adapta/pull/437/commits/2359273270bb9a549c87e26a8ec6f0429cb12a65) the Readme now shows an example of a delta lake table handling by a S3 Storage client and in order to do that I've added a new method to store the table `save_data()` I hope it's ok 👍